python/sqlflow_submitter/tensorflow/predict.py

```diff
@@ -50,35 +50,15 @@ def define_tf_flags():
 class FastPredict:
     def __init__(self, estimator, input_fn):
         self.estimator = estimator
-        self.first_run = True
-        self.closed = False
         self.input_fn = input_fn
 
-    def _create_generator(self):
-        while not self.closed:
-            yield self.next_features[0], self.next_features[1]
-
-    def predict(self, feature_batch):
-        self.next_features = feature_batch
-        if self.first_run:
-            self.batch_size = len(feature_batch)
```

Comment:
@typhoonzero BUG: the name `feature_batch` is misleading. It actually is a `(feature, label)` tuple with the batch size is always 1. However, `len(feature_batch) == 2`, and we shouldn't assign it to `self.batch_size`.